.github/workflows/slack_stale_pr.yml

```diff
@@ -0,0 +1,27 @@
+name: Find stale PRs
```

Comment:
Nitpick: Please rename to `Post Stale PRs To Slack`

Reply:
done